tests/unittest/pywbem/test_valuemapping.py

```diff
@@ -51,6 +51,16 @@ def element_kind(request):
     return request.param
 
 
+@pytest.fixture(params=[
```

Comment:
Couldn't you simplify this fixture by naming it "is_array and make the params True, False.  Not a real issue but when I figured that out it simplified other parts of the test.

Reply:
DONE